ios/brave-ios/App/iOS/Delegates/SceneDelegate.swift

```diff
@@ -480,7 +486,13 @@ extension SceneDelegate {
         break
       }
 
-      scene.browserViewController?.switchToTabForURLOrOpen(url, isPrivileged: true)
+      let isPrivateBrowsing =
+        scene.browserViewController?.privateBrowsingManager.isPrivateBrowsing == true
+      scene.browserViewController?.switchToTabForURLOrOpen(
+        url,
+        isPrivate: isPrivateBrowsing,
+        isPrivileged: true
```

Comment:
Realize it was privileged before but is there a reason this one is opened with privilege?

Reply:
:o good catch. No it should NOT be privileged. It's a custom user activity for shortcuts as per the PR it was added in: https://github.com/brave/brave-core/pull/22970/files